community/phantomjs/PKGBUILD

```diff
@@ -0,0 +1,48 @@
+# $Id$
+# Maintainer: Felix Yan <felixonmars@gmail.com>
+# Contributor: grimsock <lord.grimsock at gmail dot com>
+# Contributor: Dieter Plaetinck <dieter@plaetinck.be>
+# Contributor: Vladimir Chizhov <jagoterr@gmail.com>
+# Contributor: Henry Tang <henryykt@gmail.com>
+
+# ALARM: Kevin Mihelich <kevin@archlinuxarm.org>
+# (From qt4-4.8.4-18 PKGBUILD)
+#  - Just say NO to x86 optimizations, and no neon
+#  - added -fno-strict-volatile-bitfields to CXXFLAGS to fix ARM bug
+#  - not necessary to disable distcc because other software won't be built against phantomjs qt4
```

Comment:
it's really not necessary to say something isn't necessary, or that the notes are from another package.